_posts/2017-01-27-squad-health-checks.md

```diff
@@ -0,0 +1,70 @@
+---
+layout:     post
+title:      Squad Health Checks
+date:       2017-01-27 10:00
+summary:    Join Adele, Mark, Tom, Mark, Adam & John for our January 2017 edition of Sky Bet Tech Talks on Retrospectives, AGILE Programme Management, Discovery & Experimentation and Squad Health Checks
+category:   Tech Talks
+tags:       Tech, Talk, Agile, Squads, Health Checks
+author:     john_kitson
+---
+
+As a scrum master or agile coach, it's really important to enable a culture of continuous improvement. One way of doing this is to get the teams you work with into the habit of retrospecting. Ideally teams are always on the look-out for ways to improve their capability to meet business needs both now and in the future. Having heartbeat meetings; retrospectives where the team can discuss problems and hindrances, and ways to optimise their approach is a key part of agile working.
+
+There is no single prescribed best-practice for gauging the current state and identifying improvements, and an experienced scrum master should know the most appropriate method to use at any one time depending on the various challenges faced by the team.
+
+Methods range from lightweight and un-structured, to exhaustive and highly-structured. Examples of lightweight methods include – team temperature taking and simple games such as Sails and Anchors. Temperature taking requires each team member to submit their own personal view on a scale between 1-5, where 1 indicates there are big problems, and a 5 suggests things are going very well indeed. Sails and Anchors asks for submissions around factors that may be impeding the team, and areas that are enabling it. In each of these scenarios, discussions should abound to generate actions which may improve or optimise the situation.
```

Comment:
You don't need a dash after "include" in the second sentence. Also consider "temperature-taking" (hyphenated).